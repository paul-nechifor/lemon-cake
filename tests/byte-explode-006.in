(byte-explode :(8 0xffeeddccccddaaff) :(1 5) :(8 0xffeeddbbccddaaff))
