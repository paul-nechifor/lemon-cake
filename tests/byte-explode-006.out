(255 170 221 204 204 221 238 255 5 255 170 221 204 187 221 238 255)
